fastapi/models/fastapi_endpoint.py

```diff
@@ -200,13 +200,15 @@ def get_uid(self, root_path):
     def _get_app(self) -> FastAPI:
         app = FastAPI(**self._prepare_fastapi_app_params())
         for router in self._get_fastapi_routers():
-            app.include_router(prefix=self.root_path, router=router)
+            app.include_router(router=router)
         app.dependency_overrides[dependencies.fastapi_endpoint_id] = partial(
             lambda a: a, self.id
         )
         for exception, handler in self._get_app_exception_handlers().items():
             app.add_exception_handler(exception, handler)
-        return app
+        mainapp = FastAPI()
+        mainapp.mount(self.root_path, app)
```

Comment:
Not sure but I think that the path must be unique for each sub app you mount.

Reply:
I assume `root_path` is unique across endpoints